helper/restore_helper.go

```diff
@@ -33,41 +35,47 @@ const (
 )
 
 var (
-	contentRE *regexp.Regexp
+	writeHandle *os.File
+	writer      *bufio.Writer
+	contentRE   *regexp.Regexp
 )
 
-/* RestoreReader structure to wrap the underlying reader.
+/* IRestoreReader interface to wrap the underlying reader.
  * readerType identifies how the reader can be used
```

Comment:
Comments here are related to `RestoreReader`, but are located above `IRestoreReader`.